.github/workflows/unit-tests.yml

```diff
@@ -78,23 +110,31 @@ jobs:
         name: JS Unit Tests
         needs: Setup
         runs-on: ubuntu-latest
-
         steps:
             - uses: actions/checkout@v3
+
+            - name: Cache node_modules
+              id: cache-node-modules
+              uses: actions/cache@v3
+              env:
+                  cache-name: cache-node-modules
+              with:
+                  path: node_modules
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+
             - name: Setup node version and npm cache
               uses: actions/setup-node@v3
               with:
                   node-version-file: '.nvmrc'
                   cache: 'npm'
-            - name: Npm install and build
-              run: |
-                  npm ci --no-optional
-                  FORCE_REDUCED_MOTION=true npm run build:e2e-test
```

Comment:
Same thing here as well! We have removed the Install Node dependencies step from the `Setup` job, I think we won't need the `Cache node_modules` & `Setup node version and npm cache` steps in this case!

Reply:
See [comment above](https://github.com/woocommerce/woocommerce-blocks/pull/6404/files/6bd2555e729593a171b59d1a6b3c2037560481cf#r880346473)